src/vmm_mad/remotes/lxd/deploy

```diff
@@ -74,7 +74,9 @@ else
     begin
         operation = container.start
         raise operation if container.status != 'Running'
-    rescue LXDError => exception
+    rescue StandardError => exception
+        # TODO: Improve wait condition
+        sleep 1 # Wait for LXD to revert non-rootfs disks mountpoint
```

Comment:
As this is an exception we can be more aggressive in  the cool down of the FS and sleep for more seconds.